### ISSUE TITLE
mergeAll is capped at 1 arguments regardless of order

### ISSUE THREAD

**Issue by simonsmith** [jfmengels/eslint-plugin-lodash-fp#44]
Hello,

I'm using `mergeAll` like so:

```js
mergeAll([objA, objB], {});
```
> `mergeAll` is capped at 1 arguments and takes an array as its last argument. (lodash-fp/no-extraneous-args)

I've tried it array last but the result is an object that is merged incorrectly and the error is still present anyway.

I'm pretty sure it should be array first from my testing but I couldn't see this method documented anywhere.

Thanks

**Comment by simonsmith** [comment]
From further investigation it looks like it's supposed to accept one argument, an array. I think I misinterpreted the docs:

> Methods that accept an array as their last or only argument:
